### PR TITLE
Remove aegir, use @metamask/eslint-config

### PR DESCRIPTION
**Pull request by rekmarks**
- Remove `aegir`
- Use `@metamask/eslint-config`

### CODE REVIEW

**Comment by rekmarks** [comment]
~Currently experiencing the following lint failures:~
[redacted]

File: package.json [hunk 1]
```diff
@@ -7,16 +7,11 @@
...
-    "test:node": "aegir test --target node",
-    "test:browser": "aegir test --target browser",
-    "release": "aegir release",
-    "release-minor": "aegir release --type minor",
-    "release-major": "aegir release --type major",
-    "coverage": "aegir coverage",
-    "coverage-publish": "aegir coverage --provider coveralls"
+    "lint": "eslint . --ext js,json",
+    "lint:fix": "eslint . --ext js,json --fix",
+    "build": "./node_modules/.bin/babel src --out-dir dist",
```
Comment (whymarrh) [inline package.json]:
```suggestion
    "build": "babel src --out-dir dist",
```

**Review by whymarrh — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (rekmarks) [reply package.json]: I reckoned there had to be a better way. I don't understand why the version of the babel docs I found didn't suggest this

**Review by kumavis — APPROVED** [review]
Sure

**Pull request merged**